sdcm/utils/git.py

```diff
@@ -25,6 +30,30 @@ def get_git_current_branch() -> str:
     return proc.stdout.decode(encoding="utf-8").strip()
 
 
+def get_git_status_info() -> GitStatus:
+    try:
+        status_proc = subprocess.run(args=["git", "status", "-b", "--porcelain=v2",
+                                     ".nothing"], check=True, capture_output=True)
+        output = status_proc.stdout.decode(encoding="utf-8").strip()
+        git_status: GitStatus = {}
+        for line in [line for line in output.split("\n") if line.startswith("#")]:
```

Comment:
When doing such parsing please supply example in comments

Reply:
+1

Also, it is always better to use generator in the loop instead of the list object:
```
for line in (line for line in output.split("\n") if line.startswith("#")):
```

Reply:
Fixed